### PR TITLE
Add condition examples

### PR DESCRIPTION
**Pull request by mdroidian**
https://github.com/dvargas92495/roamjs-query-builder/blob/docs-add-examples/docs/examples.md
or a screenshot: https://github.com/dvargas92495/roamjs-query-builder/assets/3792666/4587deba-2d7c-459d-8c82-d6aed330799c

### CODE REVIEW

File: src/components/ResultsView.tsx [hunk 1]
```diff
@@ -386,6 +386,7 @@ const ResultsView: ResultsViewComponent = ({
             </Tooltip>
           )}
           <Popover
+            placement="left-end"
```
Comment (mdroidian) [inline src/components/ResultsView.tsx:389]: I also snuck this in here to deal with the placement of `Edit Query` being unclickable

image

Comment (dvargas92495) [reply src/components/ResultsView.tsx:389]: how does this behave if the qb is at the bottom of the page? I guess you could always scroll down further

**Review by dvargas92495 — APPROVED** [review]
Great stuff here 🔥

File: src/components/ResultsView.tsx (hunk 1 above)
Comment (mdroidian) [reply src/components/ResultsView.tsx:389]: In my tests, worst case scenario is `Copy Query` is hidden when there are no results.  But a user can also toggle footer to alleviate that.

https://github.com/dvargas92495/roamjs-query-builder/assets/3792666/8e405057-63a8-4064-b1c0-7e4c6b351f6e

**Pull request merged**